docs/funcx_upgrade.rst

```diff
@@ -2,46 +2,240 @@
 Upgrading from funcX SDK and funcx-endpoint
 ###########################################
 
-Note
-^^^^
-This document will be expanded at a later date to include more details.
+Overview
+^^^^^^^^
 
-Background
-^^^^^^^^^^
+The Globus team has renamed `funcX` to `Globus Compute` in order to centralize
+our infrastructure under a single umbrella.
+
+This migration involves changing the package names and most classes with
+some variation of ``FuncX`` in the name.
+
+TL;DR We recommend uninstalling the old packages and installing
+``globus-compute-sdk`` / ``globus-compute-endpoint`` instead.  Then change
+any import statements to reflect the name of the new packages.
```

Comment:
I might delineate this slightly more.  Something like:
> TL;DNR: Though it is enough to simply upgrade funcx* (e.g., ``pip install -U ...``), as of the rebrand, those packages are shims that merely import their logic from ``globus-compute-*``&nbsp;&mdash;&nbsp;we recommend migrating all scripts to use the new packages directly.  The suggested method, then, is to uninstall the funcX packages, and then install ``globus-compute-sdk`` / ``globus-compute-endpoint`` in their stead.  Accordingly, be sure to update all import statements to use the new packages.

Reply:
Huh; behooves me to finish reading.  You already address this adequately in the note.